Userland/Libraries/LibC/ar.h

```diff
@@ -0,0 +1,47 @@
+/* Header describing `ar' archive file format.
+   Copyright (C) 1996-2023 Free Software Foundation, Inc.
+   This file is part of the GNU C Library.
```

Comment:
Just to note that down somewhere, writing our own version of ar.h should be trivial if the contents are not protectable (and if they are, doing something like this is out of the question anyways).